modules/social_features/social_album/src/Plugin/views/field/SocialAlbumEntityOperations.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\social_album\Plugin\views\field;
+
+use Drupal\Core\Url;
+use Drupal\views\Plugin\views\field\EntityOperations;
+use Drupal\views\ResultRow;
+
+/**
+ * Renders all operations links for a post.
+ *
+ * @ingroup views_field_handlers
+ *
+ * @ViewsField("social_album_post_operations")
+ */
+class SocialAlbumEntityOperations extends EntityOperations {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render(ResultRow $values) {
+    foreach (get_object_vars($values) as $key => $value) {
+      if (preg_match('/_delta$/', $key)) {
+        break;
+      }
+    }
+
+    $entity = $this->getEntity($values);
+
+    return [
+      '#lazy_builder' => [
+        [$this, 'renderLinks'],
+        [
+          $values->_entity->id(),
+          $entity->id(),
+          $entity->field_post_image->get($value)->target_id,
+        ],
+      ],
+    ];
+  }
+
+  /**
+   * Lazy_builder callback; builds a post's links.
+   *
+   * @param int $node_id
+   *   The post entity ID.
+   * @param int $post_id
+   *   The post entity ID.
+   * @param int $file_id
+   *   The file entity ID.
+   *
+   * @return array
+   *   A renderable array representing the post links.
+   */
+  public static function renderLinks($node_id, $post_id, $file_id) {
+    $entity = \Drupal::entityTypeManager()->getStorage('post')->load($post_id);
```

Comment:
We can call `\Drupal::entityTypeManager()` only once here, so it's better to create a variable I think.